jetstream/resource_kv_entry.go

```diff
@@ -0,0 +1,163 @@
+package jetstream
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/nats-io/jsm.go"
+	"github.com/nats-io/nats.go"
+)
+
+func resourceKVEntry() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKVEntryCreate,
+		Read:   resourceKVEntryRead,
+		Update: resourceKVEntryUpdate,
+		Delete: resourceKVEntryDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
I wonder if its a good idea to allow import here, I imagine this can cause a ton of resources to be made?

Reply:
We often end up importing resources that we've created manually. We've even done this with some JS resources. We're planning on having a json config entry for each service we run so in our case there wouldn't be so many. We do need to update the import guide though if we're to support this.